framework/src/controller/application.js

```diff
@@ -29,7 +29,11 @@ const Controller = require('./controller');
 const version = require('../version');
 const validator = require('./validator');
 const configurator = require('./default_configurator');
-const { genesisBlockSchema, constantsSchema } = require('./schema');
+const {
+	genesisBlockSchema,
+	constantsSchema,
+	applicationConfigSchema,
```

Comment:
@pablitovicente If that's the only issue, this config schema is already been loaded to configurator, you don't need to do it here.

Reply:
Hmm will check that code then as it was not working. It was possible to specify invalid values but maybe due to the combination of wrong schema definition... will double check.

Reply:
Removed application schema validation from Application. Schema fix was enough. Resolving.